test/tools/d_do_test.d

```diff
@@ -581,7 +581,7 @@ bool gatherTestParameters(ref TestArgs testArgs, string input_dir, string input_
     findTestParameter(envData, file, "GDB_MATCH", testArgs.gdbMatch);
 
     if (findTestParameter(envData, file, "POST_SCRIPT", testArgs.postScript))
-        testArgs.postScript = replace(testArgs.postScript, "/", to!string(envData.sep));
+        {}//testArgs.postScript = replace(testArgs.postScript, "/", to!string(envData.sep));
```

Comment:
This is open for debate - not sure what should go here. The bash on my system (Windows + WSL2) gets confused.

Reply:
Could you elaborate what "confused" means? :)

Reply:
Of course:

```
+ source runnableextra-filescoverage-postscript.sh
tools/postscript.sh: line 48: runnableextra-filescoverage-postscript.sh: No such file or directory
```

Reply:
should be `runnable/extra-files/coverage/postscript.sh` so apparently all backslashes are treated as escape sequences